core-chart/templates/argocd/operator.yaml

```diff
@@ -0,0 +1,163 @@
+{{- if index .Values "argo-cd" "install" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ .Release.Name }}-argocd-kubestellar-operator"
+data:
+  # This script is used by the "{{ .Release.Name }}-argocd-kubestellar-operator" job to
+  # 1. register KubeStellar WDSes as Argo CD target clusters
+  # 2. create any Argo CD application listed in KubeStelar values
+  argocd-operator.sh: |
+    #!/bin/env bash

```

Comment:
`env` is at `/usr/bin/env`. That's the standard place. That is the only place that it is found on my Mac and in Ubuntu 22.04 and Ubuntu 20.04. That is the standard place used in the files in `scripts/`. That location is mentioned in https://man7.org/linux/man-pages/man1/env.1.html .